Modules/Devices/src/Devices.Application/Identities/Commands/CreateIdentity/Handler.cs

```diff
@@ -57,11 +52,8 @@ public async Task<CreateIdentityResponse> Handle(CreateIdentityCommand command,
 
         var user = new ApplicationUser(newIdentity);
 
-        var createUserResult = await _userManager.CreateAsync(user, command.DevicePassword);
-
-        if (!createUserResult.Succeeded)
-            throw new OperationFailedException(ApplicationErrors.Devices.RegistrationFailed(createUserResult.Errors.First().Description));
-
+        await _identityRepository.AddUserForIdentity(user, command.DevicePassword);
```

Comment:
I know it was my suggestion to call the method like that, but now that I see it it should probably just be called `AddUser`